ansible/roles_ocp_workloads/ocp4_workload_authentication/tasks/workload.yml

```diff
@@ -89,15 +100,22 @@
     loop:
     - ""
     - "HTPasswd Authentication is enabled on this cluster."
-    - "Users {{ ocp4_workload_authentication_htpasswd_user_base }}1 .. {{ ocp4_workload_authentication_htpasswd_user_base }}{{ ocp4_workload_authentication_htpasswd_user_count }} are created with password `{{ ocp4_workload_authentication_user_password }}`"
-    - "User `{{ ocp4_workload_authentication_admin_user }}` with password `{{ ocp4_workload_authentication_admin_password }}` is cluster admin."    
+    - >-
+      Users {{ ocp4_workload_authentication_htpasswd_user_base }}1 ..
+      {{ ocp4_workload_authentication_htpasswd_user_base }}{{ ocp4_workload_authentication_htpasswd_user_count }} are
```

Comment:
This is weird in the original, but would be better with string concatenation operator, `~`: 

```
{{ ocp4_workload_authentication_htpasswd_user_base ~ ocp4_workload_authentication_htpasswd_user_count }}
```